app/src/features/MovieDetailsScreen/index.tsx

```diff
@@ -44,9 +44,9 @@ const MovieDetailsScreen = (): JSX.Element => {
         const dict = movieScreenings
             //.filter((s) => s.movieStart.getTime() >= Date.now())
             .reduce((groups: any, screening: Screening): any => {
-                let day = new Date(screening.movieStart.getTime());
+                let day = new Date(screening.movieStart);
                 removeTime(day);
-                let key = day.getTime().toString();
+                let key = day.toString();
```

Comment:
sta si mi ovo menjao? pokvario se MovieDetailScreen XD

Reply:
Joj ovo je zato sto je sada movieStart tipa string, a ne Date pa sam morao posle da ga konvertujem

Reply:
Hmm onda ne znam zasto mi se lose ispisuje. Nema veze, popravicu sa moje strane, vise mi se svidja da je datum string

Reply:
To volim da cujem hahaha